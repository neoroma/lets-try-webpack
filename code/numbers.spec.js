const { one, two } = require('./numbers')

test('one returns number on', () => {
    expect(one()).toMatchSnapshot()
})

test('two returns number two', () => {
    expect(two()).toMatchSnapshot()
})