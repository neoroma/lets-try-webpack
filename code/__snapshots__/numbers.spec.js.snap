// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`one returns number on 1`] = `"number one"`;

exports[`two returns number two 1`] = `"number two"`;
